app/views/layouts/application.html.erb

```diff
@@ -44,18 +44,24 @@
           <div class="sidebar-nav navbar-collapse">
             <ul class="nav in" id="side-menu">
               <li>
-                <a href="/">
-                  <i class="fa fa-dashboard fa-fw"></i>
-                  Dashboard</a>
+                <a href="#"><i class="fa fa-list"></i> Categorias<span class="fa arrow"></span></a>
+                <ul class="nav nav-second-level collapse in">
+                  <li>
+                    <a href="<%= categories_path %>">Categorias</a>
```

Comment:
**Very important:** We have to separate `Category` by `Store`.

I created an Issue for us: https://github.com/brunoao86/easyorder/issues/12